react/src/App.js

```diff
@@ -29,6 +40,40 @@ function App() {
           onError={onError}
           onSuccess={onSuccess}
         />
+        <h2>Advanced file upload</h2>
+        <IKUpload
+          fileName="test-upload.jpg"
+          tags={["sample-tag1", "sample-tag2"]}
+          customCoordinates={"10,10,10,10"}
+          isPrivateFile={false}
+          useUniqueFileName={true}
+          responseFields={["tags"]}
+          folder={"/sample-folder"}
+          extensions={[{
+            "name": "remove-bg",
+            "options": {
+              "add_shadow": true,
+            },
+          }]}
+          webhookUrl="https://www.example.com/imagekit-webhook" // replace with your webhookUrl
+          overwriteFile={true}
+          overwriteAITags={true}
+          overwriteTags={true}
+          overwriteCustomMetadata={true}
+          customMetadata={{
```

Comment:
Comment custom metadata because it will throw error if this custom metadata field is not first created. The idea is to that this should work as it is, so commenting it will still show it to the user.